documentation/add/10_single_filebacked_operations.md

```diff
@@ -0,0 +1,77 @@
+# Intermediaries of Single Filebacked Operations
+Date: 2023-01-11
+
+## Objectives and problems to solve
+
+Currently the implementation of single-filebacked objects is based upon a temporary file beinhg created as infrequently
+as possible and modifcations to the sqw data taking place within that file. The issue with this mechanism is that while
+it may result in less data being written if operations only modify components of the data (e.g. signal/variance for many
+unary/binary ops) if a job fails or is cancelled partway through the user is left without a viable (or worse yet a
```

Comment:
```suggestion
unary/binary ops), nevertheless if a job fails or is cancelled partway through, the user is left without a viable (or worse yet a
```

`Nevertheless` is padding designed to ensure that the following if clause is clearly separated from the previous if.